src/lib/FormattedInput.tsx

```diff
@@ -54,10 +65,13 @@ const FormattedInput = <T extends FieldValues>(props: FormattedInputProps<T>) =>
                     onValueChange={(values) => {
                       onChange(values.value);
                     }}
+                    onFocus={focusHandler}
```

Comment:
Out of curiousity: Why does the focusHandler need to be added to both the `Controller` and the `PatternFormat`?

Reply:
ah sorry, my bad :-) Thanks for the clarification